setup.cfg

```diff
@@ -61,6 +61,8 @@ norecursedirs =
     *.egg-info
     .git
 testpaths = tests
+filterwarnings =
```

Comment:
We have a good number of `PendingDeprecationWarning` and `DeprecationWarning` in the test code, we need to clean them before making this change.

Reply:
All warnings should be fixed with my patch.

Reply:
OK, fixed for 3.10, let's see about the CI.